metagen_tools/tablegen.m

```diff
@@ -0,0 +1,130 @@
+
+function output = tablegen (basepath2)
+    % Get all the videos from the directory
+    file_struct = dir(fullfile(basepath2,'*.avi'));
+    %remove hidden files
+    file_struct = file_struct(~strncmp({file_struct.name}, '.', 1));
```

Comment:
Could have period in file name (though not advised). Change to make robust to various naming schemes.